apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-block-editor-details-click.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import tracksRecordEvent from './track-record-event';
+
+/**
+ * Return the event definition object to track `wpcom_block_editor_details_click`.
+ *
+ * @returns {{handler: Function, selector: string, type: string}} event object definition.
+ */
+export default () => ( {
+	selector: '.edit-post-header .table-of-contents button[aria-disabled="false"]',
```

Comment:
are we stuck selecting by the DOM here?  or is there any data action we can use?

Reply:
Yep, we are stuck with it. It's using the `Dropdown` component, which is using an internal `useState` to store the open/closed state.

https://github.com/WordPress/gutenberg/blob/164bef81386e96330128c6218a66f29c9ed8e865/packages/editor/src/components/table-of-contents/index.js

Reply:
Makes sense.

I think we will need to add an e2e test for this?

Also, this may need an `id` now as well.